app/cli/cmd/casbackend_add_azureblob.go

```diff
@@ -97,5 +102,7 @@ func newCASBackendAddAzureBlobStorageCmd() *cobra.Command {
 
 	cmd.Flags().StringVar(&container, "container", "chainloop", "Storage Container Name")
 
+	cmd.Flags().StringVar(&endpoint, "endpoint", "", "Azure Blob endpoint suffix (e.g., blob.core.windows.net, blob.core.usgovcloudapi.net)")
```

Comment:
nitpick.

Isn't `blob.core.windows.net` in fact the default and doesn't need to be provided? If so, the example provided in the flag can be somewhat misleading. 

Maybe updating the example to `Custom Azure Blob endpoint suffix (e.g., blob.core.usgovcloudapi.net), if not provided, the public Azure cloud endpoint will be used.`

Reply:
Yes, `blob.core.windows.net` is the default and will be used if the flag is not provided, I just wanted to give more than one example, but I can see how this can cause confusion. I'll update the description!